<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 255);</string>
  </property>
  <widget class="QWidget" name="side_bar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>160</width>
     <height>681</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 255);</string>
   </property>
   <widget class="QPushButton" name="patient_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>Patients</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/patientdownload .png</normaloff>../Medico_modern/patientdownload .png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="dash_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(0, 0, 0);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
border-bottom-color: rgb(0, 0, 0);
background-color: rgb(85, 255, 255);
background-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/dashboard.png</normaloff>../Medico_modern/dashboard.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="appointment_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;

border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Appointments</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/appointmentdownload (1).png</normaloff>../Medico_modern/appointmentdownload (1).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="dentist_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>Dentists</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/dentistimages.png</normaloff>../Medico_modern/dentistimages.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="payment_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>Payments</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/billingdownload (1).png</normaloff>../Medico_modern/billingdownload (1).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="expense_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>Expense</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/billingdownload (1).png</normaloff>../Medico_modern/billingdownload (1).png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
font: 75 20pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;
font: 22pt &quot;MS Shell Dlg 2&quot;;
background-color: rgba(255, 255, 255,0);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string> Medico App </string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="vname">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Visitor Name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="vname_edit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="phone">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Phone</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="phone_edit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="visit_time">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Visit Time</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="visit_time_edit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="visit_date">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Visit Date</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="visit_date_edit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="appointment_table">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>901</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <column>
    <property name="text">
     <string>Date Registered</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>First Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Phone</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Visit Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Visit time</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="visit_date_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>901</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Appointment Table</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="search_name_edit">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_btn">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
   </property>
   <property name="text">
    <string>Search Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>1121</width>
     <height>680</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Images/the-humble-co-cADflhZzgyo-unsplash.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="add_appointment_btn">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>145</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
   </property>
   <property name="text">
    <string>Add Patients</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Medico_modern/patientdownload .png</normaloff>../Medico_modern/patientdownload .png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1035</x>
     <y>120</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="visit_date_3">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Fisrt Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="visit_date_4">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Phone</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="search_phone_edit">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>side_bar</zorder>
  <zorder>vname</zorder>
  <zorder>vname_edit</zorder>
  <zorder>phone</zorder>
  <zorder>phone_edit</zorder>
  <zorder>visit_time</zorder>
  <zorder>visit_time_edit</zorder>
  <zorder>visit_date</zorder>
  <zorder>visit_date_edit</zorder>
  <zorder>appointment_table</zorder>
  <zorder>visit_date_2</zorder>
  <zorder>search_name_edit</zorder>
  <zorder>label_3</zorder>
  <zorder>add_appointment_btn</zorder>
  <zorder>label_5</zorder>
  <zorder>search_btn</zorder>
  <zorder>visit_date_3</zorder>
  <zorder>visit_date_4</zorder>
  <zorder>search_phone_edit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
